google/cloud/storage/_helpers.py

```diff
@@ -297,3 +304,52 @@ def _base64_md5hash(buffer_object):
     _write_buffer_to_hash(buffer_object, hash_obj)
     digest_bytes = hash_obj.digest()
     return base64.b64encode(digest_bytes)
+
+
+def _base64_crc32chash(buffer_object):
+    """Get crc32c hash of bytes (as base64).
+
+    :type buffer_object: bytes buffer
+    :param buffer_object: Buffer containing bytes used to compute an MD5
+                          hash (as base64).
+
+    :rtype: str
+    :returns: A base64 encoded digest of the MD5 hash.
+    """
+    hash_obj = _get_crc32c_object()
+    _write_buffer_to_hash(buffer_object, hash_obj)
+    digest_bytes = hash_obj.digest()
+    return base64.b64encode(digest_bytes)
+
+
+def _get_crc32c_object():
+    """ Get crc32c object
+    Attempt to use the Google-CRC32c package. If it isn't available, try
+    to use CRCMod. CRCMod might be using a 'slow' varietal. If so, warn...
```

Comment:
I might be confused but is CRCMod the Python version of crc32c implementation instead of the C++ library? 
Should it be called Python Native version which is slow?

Reply:
This is a bit confusing. 
crcmod -> python 2
  - contains c and a python impl
crc32c -> python 3
  - contains c and python impl

Reply:
I could reword this message maybe though?

Reply:
Oh, CRCmod is only for Python 2? That wasn't super clear to me.
Why not support both through the new crc32c? (If it's a maintenance burden then that's fine.)

Reply:
It is a large maintenance burden and requires different c extension style. Python 2 is deprecated also and I anticipate dropping support from new releases within the year.